tests/tests/constants/index.ts

```diff
@@ -12,25 +12,21 @@ export {
   FINITE_LOOP_CONTRACT_ABI,
 } from "./testContracts";
 
-export { basicTransfertx, contractCreation } from "./transactionConfigs";
-
-export const PORT = 19931;
-export const RPC_PORT = 19932;
-export const WS_PORT = 19933;
-export const SPECS_PATH = `./moonbeam-test-specs`;
-
-export const DISPLAY_LOG = process.env.MOONBEAM_LOG || false;
-export const MOONBEAM_LOG = process.env.MOONBEAM_LOG || "info";
-
-export const BINARY_PATH =
-  process.env.BINARY_PATH || `../node/standalone/target/release/moonbase-standalone`;
-export const SPAWNING_TIME = 30000;
+export {
+  PORT,
+  RPC_PORT,
+  WS_PORT,
+  SPECS_PATH,
+  DISPLAY_LOG,
+  MOONBEAM_LOG,
+  BINARY_PATH,
+  SPAWNING_TIME,
+  GENESIS_ACCOUNT,
+  GENESIS_ACCOUNT_PRIVATE_KEY,
+  TEST_ACCOUNT,
+} from "./constants";
```

Comment:
@joelamouche , I had to change your code here as it introduced a circular dependency between the index.ts and the transactionConfig.ts
It is usually bad practice to have circular dependency, here was a good exemple:
index.ts was relying on transactionConfig.ts to provide the basicTransferTx but transactionConfig.ts was relying on index.ts to retrieve the GENESIS_ACCOUNT.
Because index.ts imported transactionConfig.ts before setting the GENESIS_ACCOUNT, the basicTransferTx had "undefined" as a value for GENESIS_ACCOUNT.

Reply:
Thanks for catching this mistake